content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -42,12 +42,12 @@ The Private Terraform Enterprise Installer currently supports the following
 operating systems:
 
 * Debian 7.7+
-* Ubuntu 14.04 / 15.10 / 16.04
+* Ubuntu 14.04 / 16.04
```

Comment:
Why leave 14.04?

Reply:
My thinking was it's LTS and still in the support period. When 18.04 comes out in a few months, we'll drop support for 14.04.